Gruntfile.js

```diff
@@ -3,12 +3,28 @@ module.exports = function(grunt) {
     pkg: grunt.file.readJSON('package.json'),
     
     mochaTest: {
-      // test: {
-      //   options: {
-      //     reporter: 'spec'
-      //   },
-      //   src: ['test/**/*.js']
-      // }
+      test: {
+        options: {
+          reporter: 'spec'
+        },
+        src: [
+          'app/server/spec/**/*.js',
+          'app/client/spec/**/*.js',
+          'app/chrome/spec/**/*.js',
+        ]
+      }
+    },
+
+    mocha: {
```

Comment:
Lets keep all test in the command line. we dont need the run client tests on html